source/_components/notify.facebook.markdown

```diff
@@ -117,3 +117,87 @@ script:
                 title: Blue
                 payload: DEVELOPER_DEFINED_PAYLOAD_FOR_PICKING_BLUE
 ```
+
+You can now also use Facebook public beta broadcast API to push messages to ALL users who interacted with your chatbot on your page, without having to collect their number. This will scale to thousands of users. Facebook requires that this only be used for non-commercial purposes and they validate every message you send. Also note, your Facebook bot needs to be authorized for "page_subscritions" if you want to make it to all, but can be used right away to a selected group of testers of your choice. 
```

Comment:
Typo: `page_subscritions`

Reply:
I fixed and moved to correct branch.. Please review new version. How do I trigger new review for this?